### PR TITLE
Add dns resource

### PR DESCRIPTION
**Pull request by Anatw**

### CODE REVIEW

**Comment by Anatw** [comment]
!verified

File: ocp_resources/dns.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import Resource
+
+
+class DNS(Resource):
```
Comment (jpeimer) [inline ocp_resources/dns.py:4]: I believe we need to follow the contributing requirements:
https://github.com/RedHatQE/openshift-python-wrapper/blob/main/CONTRIBUTING.md#adding-a-new-module-resource

**Review by jpeimer — CHANGES_REQUESTED** [review]

File: ocp_resources/dns.py (hunk 1 above)
Comment (rnetser) [reply ocp_resources/dns.py:4]: iiuc this is a resource added by k8s (a user cannot add it).
please add a link to the api

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/dns.py (hunk 1 above)
Comment (Anatw) [reply ocp_resources/dns.py:4]: Ack.

**Review by myakove — APPROVED** [review]

**Pull request merged**